keps/sig-storage/1845-prioritization-on-volume-capacity/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Prioritization on Volume Capacity
+kep-number: 1845
+authors:
+  - "@cofyc"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-scheduling
+status: provisional
```

Comment:
I think this can change to implementable

Reply:
updated